src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -377,301 +124,224 @@ public void StrykerCLI_OnMutationScoreAboveThresholdBreak_ShouldReturnExitCode0(
             result.ShouldBe(0);
         }
 
-        [Fact]
-        public void StrykerCLI_WithNoFilesToExcludeSet_ShouldPassDefaultValueToStryker()
+        [Theory]
+        [InlineData("--help")]
```

Comment:
By the way, are these options reflected in the docs? I don't remember it there but I might be something here.

I mean, it's kind of obvious anyway, but for consistency...

Reply:
We could list them in the docs yes 👍